src/include/brain/operating_unit.h

```diff
@@ -69,15 +69,17 @@ class ExecutionOperatingUnitFeature {
    * @param num_keys Number of keys
    * @param cardinality Estimated cardinality
    * @param mem_factor Memory adjustment factor
+   * @param num_iterations Number of iterations
```

Comment:
Oh and similarly, please rename the `num_iterations` here and in other places to `num_loops`.